benchmark/benchmark_device_segmented_radix_sort.cpp

```diff
@@ -427,8 +427,8 @@ benchmark::RegisterBenchmark( \
 
 #define CREATE_SORT_PAIRS_DESCENDING_BENCHMARK(Key, Value, SEGMENTS) \
 benchmark::RegisterBenchmark( \
-    (std::string("sort_pairs") + "<" #Key ", " #Value ">" + \
-        "(~" + std::to_string(SEGMENTS) + " segments), descending" \
+    (std::string("sort_pairs") + "<Key Datatype:" #Key ",Value Datatype:" #Value ">" + \
```

Comment:
I've noticed some inconsistencies between with use of `Key Datatype` vs `Key Type` and idem for `Value Datatype` vs `Value Type`. Is that intentional?

Reply:
I have converted all Key/Value Type to Key/Value Datatype in most recent commits